template/web/wp-content/mu-plugins/autoload.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
This logic is handled by the [boxuk/wp-muplugin-loader](https://github.com/boxuk/wp-muplugin-loader) package, this file is not needed.